gslib/gcs_json_credentials.py

```diff
@@ -228,18 +231,18 @@ def _CheckAndGetCredentials(logger):
     # If we didn't actually try to authenticate because there were multiple
     # types of configured credentials, don't emit this warning.
     if failed_cred_type:
-      if logger.isEnabledFor(logging.DEBUG):
+      if logger and logger.isEnabledFor(logging.DEBUG):
```

Comment:
Was this giving any error? Any reason for modifying it?

Reply:
It was, but doesn't seem to be doing so anymore.  Removed